### PR TITLE
Vadim task service

### PR DESCRIPTION
**Pull request by BelovUser**
Created simple methods for TaskService

### CODE REVIEW

File: src/main/java/com/example/todoz/models/Todo.java [hunk 1]
```diff
@@ -14,6 +14,8 @@
 public class Todo {
 
     @Id
+    @GeneratedValue
+    private Long id;
     @OneToOne
     private User user;
```
Comment (dovantuan97) [inline src/main/java/com/example/todoz/models/Todo.java:20]: rename to AppUser

File: src/main/java/com/example/todoz/repos/TaskRepo.java [hunk 2]
```diff
@@ -2,6 +2,11 @@
 
 import com.example.todoz.models.Task;
 import org.springframework.data.repository.ListCrudRepository;
+import org.springframework.stereotype.Repository;
 
+import java.time.LocalDateTime;
+import java.util.List;
+@Repository
 public interface TaskRepo extends ListCrudRepository<Task, Long> {
+    List<Task> findAllByPriority(Integer priorityNumber);
```
Comment (dovantuan97) [inline src/main/java/com/example/todoz/repos/TaskRepo.java:11]: rename Integer

**Review by dovantuan97 — CHANGES_REQUESTED** [review]
just small changes and ok

**Comment by BelovUser** [comment]
Changes were made. Thanks for suggestion.

**Review by dovantuan97 — APPROVED** [review]
Everything seems ok, gg wp

**Review by Dominik-Polacek — APPROVED** [review]
GG

**Review by yenik — APPROVED** [review]
seems fine

**Review by Davias97 — APPROVED** [review]
gg

**Pull request merged**